freqtrade/optimize/backtesting.py

```diff
@@ -414,14 +414,16 @@ def _get_close_rate(self, sell_row: Tuple, trade: LocalTrade, sell: SellCheckTup
 
     def _get_adjust_trade_entry_for_candle(self, trade: LocalTrade, row: Tuple
                                            ) -> LocalTrade:
-
+        # TODO: Write tests
```

Comment:
Can you explain what this refers to?
this method is tested by `test_backtesting_adjust_position.py`. It's not directly called - but that's not really necessary, as the whole backtesting flow is tested.